src/core/symbology/qgssymbollayer.h

```diff
@@ -40,7 +41,6 @@ class QgsDxfExport;
 class QgsExpression;
 class QgsRenderContext;
 class QgsPaintEffect;
-class QgsSymbolLayerReference;
```

Comment:
Can we avoid this with an explicit destructor?

Reply:
@nyalldawson , I'm not sure, I'll try in the next run :)

Reply:
@nyalldawson , nope, that's not enough.